ext/zeek_pygments.py

```diff
@@ -1,165 +1,247 @@
-from pygments.lexer import RegexLexer, bygroups, include, words, bygroups
-from pygments.token import *
+from pygments.lexer import RegexLexer, bygroups, include, words
+from pygments.token import (
```

Comment:
Huh, this is quite cool — it actually expanded the `*`?

Reply:
No, this one I had to do manually ☹️